trestle.md

```diff
@@ -0,0 +1,262 @@
+# Adding a Admin Framework
+
+In many of our applications at Brand New Box we include a Admin portal or framework for admin users to manage standard [Create, Read, Update, and Delete](https://en.wikipedia.org/wiki/) (CRUD) operations. Right now anyone can create, update or destroy a book in our app. If you think of a traditional library, only employees could include or exclude a book from the library, so let's create similar permissions.
+
+Remember how cool it was the first time you generated scaffolding using the [rails generate](https://guides.rubyonrails.org/command_line.html#rails-generate) command? [Trestle](https://github.com/TrestleAdmin/trestle) emulates this, but creates even more features. 
+
+## Step 1 - Add the gem
+Start by opening your Gemfile: 
+```rb
+# bnb-library/Gemfile
+```
```

Comment:
With your sweet file indicator at the top of each code block, this feels redundant.

```suggestion
```